app/components/page_list_component/view.rb

```diff
@@ -17,7 +17,11 @@ def show_down_button(index)
     end
 
     def condition_description(condition)
-      I18n.t("page_conditions.condition_description", check_page_text: condition_check_page_text(condition), goto_page_text: goto_page_text_for_condition(condition), answer_value: answer_value_text_for_condition(condition))
+      if condition.secondary_skip?
+        I18n.t("page_conditions.secondary_skip_description", check_page_text: condition_check_page_text(condition), goto_page_text: goto_page_text_for_condition(condition))
+      else
+        I18n.t("page_conditions.condition_description", check_page_text: condition_check_page_text(condition), goto_page_text: goto_page_text_for_condition(condition), answer_value: answer_value_text_for_condition(condition))
+      end
```

Comment:
If we wanted to reduce lines I think we could make the logic change just the translation key?